rest-api-spec/src/main/resources/rest-api-spec/test/indices.data_stream/30_auto_create_data_stream.yml

```diff
@@ -0,0 +1,51 @@
+---
+"Put index template":
+  - skip:
+      version: " - 7.9.99"
+      reason: "not yet backported"
+      features: allowed_warnings
+
+  - do:
+      allowed_warnings:
+        - "index template [test] has index patterns [test-*] matching patterns from existing older templates [global] with patterns (global => [*]); this template [test] will take precedence during new index creation"
+      indices.put_index_template:
+        name: generic_logs_template
```

Comment:
Do we not need to remove this ITv2 at the end of the test?

Reply:
V2 templates are removed at the end of each test automatically, so shouldn't be a concern